pkg/volume/local/local.go

```diff
@@ -208,6 +207,28 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return nil
 	}
 
+        oldMount, err := mount.IsAlreadyMounted(dir)
```

Comment:
not sure about that, from dir I extract only pvc/volume name. Which is used to check if it is mounted already.